regulations/static/regulations/js/source/views/main/search-results-view.js

```diff
@@ -39,6 +39,8 @@ var SearchResultsView = ChildView.extend({
             this.url = 'search/' + this.options.id;
 
             ChildView.prototype.initialize.apply(this, arguments);
+        } else {
+            this.options.docType = this.$el.data('doc-type');
```

Comment:
I think the `render` flag is confusingly, but it's `true` on pjax load and `false` when the expected content has already been loaded. The relevant content is only populated at this point on `false`, so otherwise `this.$el` will resolve to the wrong element, or nothing, and we can't use it to look up `docType`.